src/applications/vaos/appointment-list/components/ScheduleNewAppointmentRadioButtons.jsx

```diff
@@ -75,9 +77,10 @@ function ScheduleNewAppointmentRadioButtons({
 
   return (
     <div className="vads-u-padding-y--3 vads-u-margin-bottom--3 vads-u-border-top--1px vads-u-border-bottom--1px vads-u-border-color--gray-lighter">
-      <h2 className="vads-u-font-size--h3 vads-u-padding-bottom--0 vads-u-margin-y--0">
+      <h2 className="vads-u-padding-bottom--0 vads-u-margin-y--0">
         Schedule a new appointment
       </h2>
+      <div>Choose an appointment type.</div>
```

Comment:
I think this text is meant to the be the label/legend for the radio buttons. Instead of putting it here, I would adjust the text in the `label` prop of `RadioButtons` down below to remove the `sr-only` class and update the text to match what you have here.

Reply:
Good catch. Change made.